chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -301,6 +301,13 @@
   notifies :restart, "service[chef-solr]", :delayed
 end
 
+# Make systemd restart chef services (and their deps) on failure
+utils_systemd_service_restart "rabbitmq-server"
```

Comment:
Won't this crowbar dependency for rabbitmq overlap with the one with use for the cluster?

Reply:
No, that's the rabbitmq on the admin server, used by chef. It's completely independent of anything we do with rabbitmq for openstack.